### PR TITLE
kubelet logs print 'kubelet nodes sync' frequently

### PR DESCRIPTION
**Pull request by chymy**
Signed-off-by: chymy <chang.min1@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
`kubeInformers.Core().V1().Nodes().Informer().HasSynced()` returns true, kubelet logs print `kubelet nodes sync` frequently.
```
...
I0119 00:07:50.745078   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:50.845316   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:50.945620   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.045768   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.145950   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.246268   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.346477   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.446598   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.546733   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.646869   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.747062   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.847257   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:51.947565   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.047724   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.147870   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.247981   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.348379   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.448427   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.548691   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.649026   13507 kubelet.go:447] kubelet nodes sync
I0119 00:07:52.749439   13507 kubelet.go:447] kubelet nodes sync
...
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -444,7 +444,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		nodeLister = kubeInformers.Core().V1().Nodes().Lister()
 		nodeHasSynced = func() bool {
 			if kubeInformers.Core().V1().Nodes().Informer().HasSynced() {
-				klog.Infof("kubelet nodes sync")
+				klog.V(1).Infof("kubelet nodes sync")
```
Comment (ehashman) [inline pkg/kubelet/kubelet.go:447]: I'm not sure if V(1) is the right log level for this but WFM

**Review by ehashman — APPROVED** [review]
/ok-to-test
/lgtm
/priority important-soon

/cc @derekwaynecarr
this logspam came in from the changes in #94087

**Comment by ehashman** [comment]
Good catch, thanks @chymy !

File: pkg/kubelet/kubelet.go (hunk 1 above)
Comment (sjenning) [reply pkg/kubelet/kubelet.go:447]: Lets just remove this line completely.  It is the expected case and doesn't offer a lot of debugging value.  The line with debugging value is 450 indicating the informer has not yet synced.  Thans!

Comment (chymy) [reply pkg/kubelet/kubelet.go]: Thanks for your reviews @sjenning @ehashman , fixed.

**Review by ehashman — APPROVED** [review]
/lgtm

thanks for the update

**Comment by ehashman** [comment]
@sjenning can you approve?

**Comment by rphillips** [comment]
+1 lgtm

**Comment by alculquicondor** [comment]
Should we backport this?

**Comment by ehashman** [comment]
@alculquicondor we backported it at the same time as #94087, I held those PRs to grab them together.

**Comment by alculquicondor** [comment]
awesome, thanks

**Pull request merged**